lib/iris/experimental/ugrid.py

```diff
@@ -2927,7 +2931,13 @@ def _string_summary(self, repr_style):
         )
         # Add 'other' metadata that is drawn from the underlying node-coord.
         # But put these *afterward*, unlike other similar classes.
-        for item in ("standard_name", "units", "long_name", "attributes"):
+        for item in (
+            "shape",
```

Comment:
@pp-mo Snuck in `shape` here... no objections, just curious as to the reasoning

Reply:
Well it seems clear that when printing these mesh information components, we don't want to realise or print out contents.
But when debugging I saw the printouts, it seemed odd you couldn't even see the shape., unlike dask arrays which show the shape + dtype.  
So I added it in (but not dtype).  

Since then I realised that you + @trexfeathers had already agreed a printout format for Connectivity, which does not include shape.  
So I now think that we should probably at least make those 2 the same :  
**What do you think + which way do you reckon we should we jump ?**

Reply:
@pp-mo Awesome, thanks.

I was going to re-haul how coordinates in general do `__str__` and `__repr__` in order to keep everything lazy.

I think there is real tangible benefit there, but outside the scope of this PR